docs: 7.6.1 MEND_RNV_GITHUB_BOT_USER_ID (#547)
mend/renovate-ce-ee: docs/configuration-options.md

**`RNV_GITHUB_PEM_FILE_PATH`**: [GitHub only] The file path for GitHub app key. Defaults to `/usr/src/app/renovate.private-key.pem`.

**`MEND_RNV_GITHUB_BOT_USER_ID`**: [GitHub only] Optional: the bot user ID that will be used in `gitAuthor` (e.g. `myBotName[bot] <123456+myBotName[bot]@users.noreply.github.com`), the value can be found by calling `https://api.github.com/users/{appName}[bot]` under the `id` key (replace the `{appName}` with the actual app name).

> Note: By default Renovate server will attemp to call this endpoint once during startup (both CE and EE server) and it does not require authentication. if you wish to skip this call for any reason you will need to provide the value in `MEND_RNV_GITHUB_BOT_USER_ID=<id-number>`

Reviewer: typo : attemp => attempt

> **`MEND_RNV_GITHUB_BOT_USER_ID`**: [GitHub only] Optional: the bot user ID that will be used in `gitAuthor` (exampe auther `myBotName[bot] <123456+myBotName[bot]@users.noreply.github.com` and the user id is `123456`), the value can be found by calling `https://api.github.com/users/{appName}[bot]` under the `id` key (replace the `{appName}` with the actual app name).
> Note: By default Renovate server will attempt to call this endpoint once during startup (both CE and EE server) and it does not require authentication. if you wish to skip this call for any reason you will need to provide the value in `MEND_RNV_GITHUB_BOT_USER_ID=<id-number>`

Reviewer: ```suggestion
**`MEND_RNV_GITHUB_BOT_USER_ID`**: [GitHub only] Optional: The bot user ID that will be used in `gitAuthor` (example author `myBotName[bot] <123456+myBotName[bot]@users.noreply.github.com` and the user id is `123456`). The value can be found by calling `https://api.github.com/users/{appName}[bot]` under the `id` key (replace the `{appName}` with the actual app name).
Note: By default Renovate server will attempt to call this endpoint once during startup (both CE and EE server) and it does not require authentication. If you wish to skip this call for any reason you will need to provide the value in `MEND_RNV_GITHUB_BOT_USER_ID=<id-number>`
```
